src/server/src/main/java/io/cassandrareaper/jmx/CompactionProxy.java

```diff
@@ -93,4 +94,14 @@ public void forceCompaction(String keyspaceName, String... tableNames) {
     }
     return activeCompactions;
   }
+
+  public Integer getPendingCompactions() {
+    try {
+      return proxy.getPendingCompactions();
+    } catch (JMException e) {
+      LOG.warn(String.format("Could not fetch pending compactions from %s", proxy.getHost()));
```

Comment:
Could you add the exception as argument to `LOG.warn` so we get the stack trace in the logs?
This should do it (you don't need `String.format()` either):

```
LOG.warn("Could not fetch pending compactions from {}", proxy.getHost(), e);
```

Reply:
👍